keylime/cloud_verifier_tornado.py

```diff
@@ -1005,21 +1085,24 @@ async def process_agent(agent, new_operational_state, failure=Failure(Component.
         except SQLAlchemyError as e:
             logger.error("SQLAlchemy Error: %s", e)
 
+        # Load agent's IMA policy
+        ima_policy = stored_agent.ima_policy
```

Comment:
Policies can get quite big, maybe in the future we can cache the policies loaded in a process, but for now this is comparable to the old behavior.